mysql-test/suite/rpl/t/bug84415-slave.opt

```diff
@@ -0,0 +1 @@
+--slave-transaction-retries=0
```

Comment:
is there a way to set `slave-transaction-retries` dynamically to get rid of `.opt` file?